install/connections.csv

```diff
@@ -1057,6 +1057,8 @@ WKIO,10,Waikaia,Portable,,2014-03-19T00:50:00Z,2014-09-14T00:00:00Z
 WKTO,10,Waikouaiti,Portable,,2014-09-17T03:10:00Z,2015-04-15T07:00:00Z
 WKZ,10,Branch Creek Road,,0,2004-06-03T00:00:00Z,9999-01-01T00:00:00Z
 WKZ,20,Branch Creek Road,,3,2004-06-03T00:00:00Z,2017-06-28T20:20:00Z
+WLCZ,10,Wellington College,,,2024-10-16T03:30:00Z,9999-01-01T00:00:00Z
+WLCZ,21,Wellington College,,,2024-10-16T03:30:00Z,9999-01-01T00:00:00Z
```

Comment:
is there a need to fill up the role for broadband and for strong motion?

Reply:
No need for roles since both sensors are connected to the same datalogger. Roles are only used to distinguish between different dataloggers at the same place